packages/lesswrong/components/Layout.tsx

```diff
@@ -4,7 +4,7 @@ import { useUpdate } from '../lib/crud/withUpdate';
 import { Helmet } from 'react-helmet';
 import classNames from 'classnames'
 import { useTheme } from './themes/useTheme';
-import { useLocation } from '../lib/routeUtil';
+import { subforumSlugsSetting, useLocation } from '../lib/routeUtil';
```

Comment:
Having a database setting for the subforum slugs is to avoid having to refetch the tag multiple times to check if it's a subforum. I would like to get rid of this fairly quickly